data/tests/fwupdtool.sh

```diff
@@ -35,11 +35,38 @@ echo "Show version output"
 run --version
 rc=$?; if [ $rc != 0 ]; then error $rc; fi
 
+# ---
+echo "Show version output (json)"
+run --version --json
+rc=$?; if [ $rc != 0 ]; then error $rc; fi
+
 # ---
 echo "Showing hwids"
 run hwids
 rc=$?; if [ $rc != 0 ]; then error $rc; fi
 
+if [ -e /sys/firmware/efi/efivars/BootOrder-8be4df61-93ca-11d2-aa0d-00e098032b8c ]; then
+       # ---
+       echo "Showing EFI boot"
+       run efivar-boot
+       rc=$?; if [ $rc != 0 ]; then error $rc; fi
+
+       # ---
+       echo "Showing EFI files"
+       run efivar-files
+       rc=$?; if [ $rc != 0 ]; then error $rc; fi
+
+       # ---
+       echo "Showing EFI variables"
+       run efivar-list 8be4df61-93ca-11d2-aa0d-00e098032b8c
+       rc=$?; if [ $rc != 0 ]; then error $rc; fi
+fi
```

Comment:
I don't really like relying upon the platform for this test.  Could we instead have a local test variable and override the `FWUPD_*` variables to set the path to use it instead?

Reply:
I think we need something much more involved really; I'll get onto something that can setup BootOrder, BootNext, BootXXXX etc so that we can test with some fake data. Okay if I merge the other commits to master and leave this one on this branch?

Reply:
Okay, rebased and force pushed -- lets see what CI makes of **that**.